app/model.py

```diff
@@ -29,9 +29,9 @@ def __call__(self, n_matches: int, profile_id: str) -> List[str]:
 
         def sort_mentors(mentor: Dict) -> Tuple:
             return (
-                mentee["experience_level"] != mentor["experience_level"],
                 mentee["pair_programming"] != mentor["pair_programming"],
-                mentor["industry_knowledge"],
+                mentee["job_help"] != mentor["job_help"],
+                mentee["industry_knowledge"] != mentor["industry_knowledge"],
```

Comment:
Mentee industry knowledge isn't a thing. We'll need to rethink this bit.

Reply:
It may be a good idea to change the name of this field to something like "seeking_industry_knowledge" or "industry_coaching" to avoid confusion